src/shared/helpers/browserSupport.ts

```diff
@@ -26,7 +26,11 @@ const hasOutdatedMediaQueryList = () => {
   );
 };
 
-const browserChecks = [hasOutdatedMediaQueryList];
+const abortControllerSupport = () => {
+  return !('AbortController' in window);
```

Comment:
why `!`?

Reply:
Because the check in the code want the function to return true to show the unsupported-browser page. So when abortcontroller in window is not present for old browser it will return false but we want it to return true. I hope i understood this right :) 

```const browserChecks = [hasOutdatedMediaQueryList, abortControllerSupport];

const ensureBrowserSupport = () => {
  if (browserChecks.some((check) => check())) {
    window.location.replace('/unsupported-browser');
  }
};```